src/NuGet.Clients/NuGet.CommandLine/Commands/ProjectFactory.cs

```diff
@@ -482,9 +478,7 @@ private string ResolveTargetPath()
             if (string.IsNullOrEmpty(targetPath))
             {
                 string outputPath = _project.GetPropertyValue("OutputPath");
-                string configuration = _project.GetPropertyValue("Configuration");
-                string projectName = Path.GetFileName(Path.GetDirectoryName(_project.FullPath));
-                targetPath = PathUtility.EnsureTrailingSlash(Path.Combine(outputPath, projectName, "bin", configuration));
+                targetPath = PathUtility.EnsureTrailingSlash(Path.Combine(_project.FullPath, outputPath));
```

Comment:
Can you elaborate here? 
I couldn't follow why we're removing the Configuration.

Reply:
`OutputPath` may contain `Configuration` and others already. So, we don't need to append them again.

Reply:
Do we have existing tests that validate this scenario? 
I wouldn't be surprised if no is the answer. 
I would just be hesitant to change without a test footprint.

Reply:
I don't think so. That's why I need CI to validate this patch. If it passes then there'll be no specific test cases but if it doesn't then I'll know why those tests fail.